packages/fast-components-styles-msft/src/progress/index.ts

```diff
@@ -0,0 +1,109 @@
+import designSystemDefaults, { IDesignSystem } from "../design-system";
+import { ComponentStyles, ComponentStyleSheet, ICSSRules } from "@microsoft/fast-jss-manager";
+import { applyLocalizedProperty, Direction, toPx } from "@microsoft/fast-jss-utilities";
+import { IMSFTProgressClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import Chroma from "chroma-js";
+import { get } from "lodash";
+
+/* tslint:disable-next-line */
+const styles: ComponentStyles<IMSFTProgressClassNameContract, IDesignSystem> = (config: IDesignSystem): ComponentStyleSheet<IMSFTProgressClassNameContract, IDesignSystem> => {
+    const backgroundColor: string = get(config, "backgroundColor") || designSystemDefaults.backgroundColor;
+    const brandColor: string = get(config, "brandColor") || designSystemDefaults.brandColor;
+    const direction: Direction = get(config, "direction") || designSystemDefaults.direction;
+    const foregroundColor: string = get(config, "foregroundColor") || designSystemDefaults.foregroundColor;
+
+    return {
+        progress: {
+            display: "flex",
+            width: "100%",
+            alignItems: "center",
+            height: "10px"
+        },
+        progress_indicator__determinate: {
+            display: "flex",

```

Comment:
You've got some common styles between `progress_indicator__determinate` and `progress_indicator__indeterminate` - I would add a base class (`progress_indicator`) that gets applied to both containers that apply the common styles.